src/site/content/en/blog/pwa-url-handler/index.md

```diff
@@ -0,0 +1,291 @@
+
+### Suggested use cases for PWAs as URL Handlers {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Music or video streaming sites so track links or playlist links open in the player experience of
+  the app.
+- News or RSS readers so followed or subscribed-to sites open in the app's reader mode experience.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use PWAs as URL Handlers {: #use }
+
+### Enabling via about://flags
+
+To experiment with PWAs as URL Handlers locally, without an origin trial token, enable the
+`#enable-desktop-pwas-url-handling` flag in `about://flags`.
+
+{% Aside %} Support for the PWAs as URL Handlers feature is limited to desktop operating systems
+like Window, macOS, and Linux. On Android, Chromium browsers install PWAs by generating a
+[WebAPK](https://developers.google.com/web/fundamentals/integration/webapks), which can register a
+set of intent filters for all URLs within the scope of the app. This means that PWAs already handle
+associated URLs on Android at the operating system level using intent filters. {% endAside %}
+
+### The `"url_handlers"` Web App Manifest member
+
+To associate an installed PWA with URL patterns, these patterns need to be specified in the Web App
+Manifest. This happens through the `"url_handlers"` member. It accepts an array of objects with the
+following property:
+
+- `origin`: A required `string`, which is a pattern for matching origins. These patterns are allowed
+  to have a wildcard (`*`) prefix in order to include multiple sub-domains (like `*.example.com`).
+  URLs that match these origins could be handled by this web app. The scheme is always assumed to be
+  `https://`.
+
+The excerpt of a Web App Manifest below shows how the music PWA example from the introductory
+paragraph could set this up. The second entry with the wildcard (`"*.music.example.com"`) makes sure
+that the app also gets activated for `www.music.example.com` or potential other examples like
+`marketing-activity.music.example.com`.
+
+```json
+{
+  "url_handlers": [
+    {
+      "origin": "music.example.com"
+    },
+    {
+      "origin": "*.music.example.com"
+    },
+    {
+      "origin": "🎵.example.com"
+    }
+  ]
+}
+```
+
+{%Aside %} While in an online scenario short links from `🎵.example.com` would typically be
+redirected to `music.example.com`, such navigation redirection is not a good alternative with
```

Comment:
```suggestion
redirected to `music.example.com`. Such navigation redirection is not a good alternative with
```

Reply:
Why isn't it a good alternative? Seems like you're leaving something out.

Reply:
Because the redirect can't be followed if the app is offline, but the link handling could still kick in and the app could see that it has the linked item (like a song) in its offline cache.